tests/e2e/cloudprovider/test-lb-service.sh

```diff
@@ -101,7 +101,7 @@ function wait_for_service_address {
 
   end=$(($(date +%s) + ${TIMEOUT}))
   while true; do
-    ipaddr=$(kubectl -n $NAMESPACE describe service ${service_name} | grep 'LoadBalancer Ingress' | awk -F":" '{print $2}' | tr -d ' ')
+    ipaddr=$(kubectl -n $NAMESPACE describe service ${service_name} | grep 'LoadBalancer Ingress' | awk '{print $3}')
```

Comment:
the issue with this is, that starting from kubernetes 1.31 there is now extra information (VIP) or (Proxy) in the same line

Reply:
good catch ,thanks for the PR